README.md

```diff
@@ -70,20 +70,21 @@ T_w_i = sessions.trajectories[keys[0]]  # first pose, from sensor/rig to world
 
 - Python >= 3.8
 - [hloc](https://github.com/cvg/Hierarchical-Localization) and its dependencies, including [COLMAP](https://colmap.github.io/install.html) built from source
-- everything listed in `requirements/lamar.txt` installed with
+
+:two: Install the LaMAR libraries and pull the remaining pip dependencies:
 ```bash
-python -m pip install -r requirements/lamar.txt
+python -m pip install -e .
```

Comment:
Isn't it doing more than that (installing everything listed in `requirements/lamar.txt`)? Isn't this install it in development mode with the `-e`?

Reply:
1. Installing the package now automatically pulls the dependencies. This makes it possible to run `pip install git+https://github.com/microsoft/lamar-benchmark.git` when we don't want a local copy.
2. `-e` enables editable mode (needed if the users need to edit the code, most likely if extending it).